mora/service/employee.py

```diff
@@ -413,6 +415,12 @@ def create_employee(employee_uuid):
 
         handler(str(employee_uuid), req)
 
+        # Write a noop entry to the user, to be used for the history
+        common.add_bruger_history_entry(
+            employee_uuid,
+            "Opret {}".format(common.RELATION_TRANSLATIONS[role_type])
```

Comment:
I think we should consider making the notes even more verbose and descriptive — but not necessarily right now.

Reply:
Ideally we would add specific information regarding what was changed, but that probably requires rewriting a bunch of logic, as that information is not readily available.